MAUI/Toolbar/toolbar-items.md

```diff
@@ -0,0 +1,1903 @@
+                    <toolbar:SfToolbarItem.Icon>
+                        <FontImageSource Glyph="&#xE751;"
+                                FontFamily="MauiMaterialAssets" />
+                    </toolbar:SfToolbarItem.Icon>
+                </toolbar:SfToolbarItem>
+                <toolbar:SfToolbarItem Name="AlignRight">
+                    <toolbar:SfToolbarItem.Icon>
+                        <FontImageSource Glyph="&#xE753;"
+                    FontFamily="MauiMaterialAssets" />
+                    </toolbar:SfToolbarItem.Icon>
+                </toolbar:SfToolbarItem>
+                <toolbar:SfToolbarItem Name="AlignCenter">
+                    <toolbar:SfToolbarItem.Icon>
+                        <FontImageSource Glyph="&#xE752;"
+                    FontFamily="MauiMaterialAssets" />
+                    </toolbar:SfToolbarItem.Icon>
+                </toolbar:SfToolbarItem>
+                <toolbar:SfToolbarItem Name="AlignJustify">
+                    <toolbar:SfToolbarItem.Icon>
+                        <FontImageSource Glyph="&#xE74F;"
+                    FontFamily="MauiMaterialAssets" />
+                    </toolbar:SfToolbarItem.Icon>
+                </toolbar:SfToolbarItem>
+            </toolbar:SfToolbar.Items>
+        </toolbar:SfToolbar>
+    </Grid>
+</ContentPage>
+
+{% endhighlight %}
+
+{% highlight c# %}
+
+using Syncfusion.Maui.Toolbar;
+
+namespace ToolbarSample
+{
+    public partial class MainPage : ContentPage
+    {
+        public MainPage()
+        {
+            InitializeComponent();
+
+            SfToolbar toolbar = new SfToolbar();
+            toolbar.HeightRequest = 56;
+
+            ObservableCollection<BaseToolbarItem> toolbarItems = new ObservableCollection<BaseToolbarItem>
+            {
+                new SfToolbarItem
+                {
+                    Name = "AlignLeft",
+                    Icon = new FontImageSource { Glyph = "\uE751", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "AlignRight",
+                    Icon = new FontImageSource { Glyph = "\uE753", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "AlignCenter",
+                    Icon = new FontImageSource { Glyph = "\uE752", FontFamily = "MauiMaterialAssets" }
+                },
+                new SfToolbarItem
+                {
+                    Name = "AlignJustify",
+                    Icon = new FontImageSource { Glyph = "\uE74F", FontFamily = "MauiMaterialAssets" }
+                }
+            };
+
+            toolbar.Items = toolbarItems;
+            this.Content = toolbar;
+        }
+    }
+}
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## Separator View
+
+The control provides an option to add a separator view between the toolbar items using the `SeparatorView` property.
```

Comment:
The control allows you to add a separator view using the `SeparatorToolbarItem` in the `Items` property of SfToolbar.